app/views/sections/show_events.html.slim

```diff
@@ -0,0 +1,15 @@
+.sections-show-container
+  .hero
+    .grid-container
+      .title.text-center
+        .grid-x.grid-padding-x.align-middle
+          .auto.cell
+          .shrink.cell
+            .img-container
+              = image_tag @section.image_url
+          .shrink.cell
+            h2 = @section.name
```

Comment:
Same here about the space before the `=` ;) I'll stop commenting, but please change them all ;)